dotnet-datetime/StopWatchCSharp/StopWatchCSharpTests/StopWatchUnitTests.cs

```diff
@@ -0,0 +1,72 @@
+using System.Diagnostics;
+
+namespace StopWatchCSharpTests
+{
+    [TestClass]
+    public class StopWatchUnitTests
+    {
+        private readonly int[] _numArray;
+        private readonly StopWatchMethods _stopWatchMethods = new();
+
+        public StopWatchUnitTests() 
+        {
+            _numArray = _stopWatchMethods.CreateRandomArray(20000);
+        }
+
+        [TestMethod]
+        public void GivenBubbleSortAlgorithm_WhenInvoked_VerifyStopwatchSuccess()
+        {
+            var stopWatch = _stopWatchMethods.BubbleSort(_numArray, _numArray.Length);
+
+            Assert.IsInstanceOfType(stopWatch, typeof(Stopwatch));
+            Assert.IsInstanceOfType(stopWatch.Elapsed, typeof(TimeSpan));
+            Assert.IsFalse(stopWatch.IsRunning);
+
+            stopWatch.Reset();
+
+            Assert.AreEqual(stopWatch.Elapsed, TimeSpan.Zero);
+        }
+
+        [TestMethod]
+        public void GivenHeapSortAlgorithm_WhenInvoked_VerifyStopwatchSuccess()
```

Comment:
Given that the name of this test method is similar to the previous method, one would expect the assertions and operations to be similar, so we can remove the restarting of the stopwatch and other operations. Basically since they both are testing that a stopwatch with some kind of elapsed time, that is no longer running, was returned from the sort method, the assertions should be focused on that in both cases.